src/setupTests.js

```diff
@@ -1,4 +1,29 @@
-import { configure } from "enzyme";
+/* eslint-disable import/prefer-default-export */
+import React from "react";
+import { configure, mount } from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
+import { Provider } from "react-redux";
+import { ConnectedRouter } from "connected-react-router";
+
+// import setupBase from "./setupBase";
```

Comment:
Please remove this line of commented code if it is not in use

Reply:
My apologies. This line is a piece of code I will want to try to implement soon for testing after this week's sprint. It was intentional.